config.go

```diff
@@ -623,6 +624,15 @@ func SortingWriterConfig(options ...SortingOption) WriterOption {
 	return writerOption(func(config *WriterConfig) { config.Sorting.Apply(options...) })
 }
 
+// SkipBounds lists the path to a column that shouldn't have bounds written to the
+// footer of the parquet file. This is useful for data blobs, like a raw html file,
+// where the bounds are not meaningful.
+//
+// This option is additive, it may be used multiple times to skip multiple columns.
+func SkipBounds(path ...string) WriterOption {
```

Comment:
What do you think about naming this `SkipPageBounds` to be more specific and consistent with `SkipPageIndex`?

Reply:
great